apptools/preferences/ui/widget_editor.py

```diff
@@ -7,7 +7,8 @@
 from traitsui.api import EditorFactory
 
 # fixme: We need to import the 'Editor' class from the appropriate toolkit.
-exec('from traitsui.%s.editor import Editor' % ETSConfig.toolkit)
+toolkit = {'wx': 'wx', 'qt4': 'qt4', 'qt': 'qt4'}[ETSConfig.toolkit]
+exec(f'from traitsui.{toolkit}.editor import Editor')
```

Comment:
The right way to do this is to get the toolkit object from traitsui, and ask for the editor, something like:
```suggestion
from traitsui.toolkit import toolkit_object
Editor = toolkit_object('editor.Editor')
```